src/components/CommentList/CommentList.jsx

```diff
@@ -2,0 +1,10 @@
+import { CommentInfo } from '../CommentInfo';
+import './CommentList.scss';
+
+export const CommentList = ({ comments }) => (
+  <div className="CommentList">
+    {comments.map(comment => (
+      <CommentInfo comment={comment} key={comment.id} />
```

Comment:
The `key` prop is correctly used here with `comment.id`. Ensure that each comment object has a unique `id` field to prevent rendering issues.